os/kernel/debug/sysdbg.c

```diff
@@ -416,60 +326,86 @@ static void sysdbg_print(void)
 
 static ssize_t sysdbg_write(FAR struct file *filep, FAR const char *buffer, size_t buflen)
 {
-	char arg1[32] = { '\0' };
-	char arg2[32] = { '\0' };
+	char option[32] = { '\0' };
+	char count[32] = { '\0' };
+
+	sscanf(buffer, "%s %s", option, count);
```

Comment:
If user inputs characters over 32, then is it ok? no overflow of the arrary? Could you confirm?

Reply:
The check is handled by the sysdbgapp in general.